lib_common.gradle

```diff
@@ -34,8 +34,6 @@ abstract class LibraryVersionFormatter {
                 switch (branch) {
                     case "develop":
                         return new DevelopVersionFormatter()
-                    case "HEAD":
-                        return new HeadVersionFormatter()

```

Comment:
This was necessary to allow the library to build on a HEAD commit, which was the point of the bug. Without it, you may be able to do a Gradle sync, but you can't build, which breaks rebase --exec, bisect, etc.

You can test this by using your build script locally in mocom-sdk. I just did the following:

```diff
-apply from: "https://www.github.com/geotracsystems/android-build-scripts/raw/1.3/lib_common.gradle"
+apply from: "/home/jkeys/Development/in-house/android/libraries/build-scripts/lib_common.gradle"

Reply:
Yes. I have tested it. Because `getPublishUrl` returns empty URL in HEAD branch, `fromBranch`is not reached in this case.